# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/alleyoop/symtab-newbinutils.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/symtab.c.orig	2004-09-07 00:09:57.175564888 +0200
+++ ./src/symtab.c	2004-09-07 00:10:11.678360128 +0200
@@ -115,7 +115,7 @@
 	
 	map->text_section = section;
 	map->text_start = load_addr + bfd_section_vma (map->abfd, section);
-	map->text_end = map->text_start + bfd_get_section_size_before_reloc (section);
+	map->text_end = map->text_start + bfd_get_section_size (section);
 	
 	map->filename = g_strdup (filename);
 	map->libname = g_strdup (libname);
